src/components/Waveform/Waveform.tsx

```diff
@@ -500,18 +509,17 @@ export const Waveform = forwardRef<IWaveformRef, IWaveform>((props, ref) => {
       <View
         ref={viewRef}
         style={styles.waveformInnerContainer}
-        onLayout={() => {
-          viewRef.current?.measure((_x, _y, width, height, pageX, pageY) => {
-            setViewLayout({ height, width, x: pageX, y: pageY });
-          });
+        onLayout={(event: LayoutChangeEvent) => {
+          const { height, width, x, y } = event.nativeEvent.layout;
```

Comment:
@nilesh-simform have you tested seeking works properly after this change ?

Reply:
Yes, it works fine in both android and ios